### PR TITLE
Orion conversion syringe in-game

### PR DESCRIPTION
**Pull request by Aroliacue**
:cl: Aroliacue
rscadd: Makes the Orion Conversion Syringe purchasable via cargo and adds one to the preset Orion organs in ONI base.
tweak: Trade value of mined ores and smelted bars reduced.
/:cl:

### CODE REVIEW

File: code/modules/halo/unsc/supply/onispecial.dm [hunk 1]
```diff
@@ -18,7 +18,8 @@
 	/obj/item/organ/internal/eyes/occipital_reversal/theta = 1,
 	/obj/item/organ/internal/heart/spartan/theta = 1,
 	/obj/item/organ/internal/liver/spartan/theta = 1,
-	/obj/item/organ/internal/lungs/theta = 1
+	/obj/item/organ/internal/lungs/theta = 1,
+	/obj/item/species_convert/orion = 1
 	)
 	cost = 3500
```
Comment (X0-11) [inline code/modules/halo/unsc/supply/onispecial.dm:24]: Did you re-lower those increased prices you added afterwards?

Comment (Aroliacue) [reply code/modules/halo/unsc/supply/onispecial.dm:24]: What should I lower the prices to?

Comment (X0-11) [reply code/modules/halo/unsc/supply/onispecial.dm:24]: The materials sales prices, remember? I swear you said you were going to re-nerf them after realising the sheer amount that processing ores and stuff gave you for short term work.

Comment (Aroliacue) [reply code/modules/halo/unsc/supply/onispecial.dm:24]: OH RIGHT FUCK

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**